Neos.ContentRepository.Export/src/Event/ValueObject/ExportedEvent.php

```diff
@@ -28,11 +28,14 @@ public static function fromRawEvent(Event $event): self
         // unset content stream id as this is overwritten during import
         unset($payload['contentStreamId'], $payload['workspaceName']);
 
+        $metaData = $event->metadata?->value ?? [];
+        unset($metaData['commandClass'], $metaData['commandPayload']);
+
         return new self(
             $event->id->value,
             $event->type->value,
             $payload,
-            $event->metadata?->value ?? [],
+            $metaData,
```

Comment:
**_DISCUSSION: WHY do we export the metadata at all?_**

Reply:
we seem to have done this since ever: https://github.com/neos/neos-development-collection/commit/eaf246cd313b4650fab749913ea9ac1de858acbd#diff-6c3b4d67660db1b5b8abed20f90f1018dbc58d0d2b970b0ad34cd75c7f7df855R139